.github/ISSUE_TEMPLATE/maintenance-issue.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Maintenance issue
```

Comment:
should maintenance and CI/CDe be separate? I think they tend to be highly intersectional. Since both issue types are also rare, why not just merge then?

Reply:
I had gone back and forth about it. I'm fine combining them.